test/commands/scheduled/recently-published.test.ts

```diff
@@ -81,7 +81,7 @@ describe("scheduled:recently-published", () => {
           text: {
             type: "mrkdwn",
             text:
-              "> <p>Steve Hicks and Justin Bennett talk about empathy in workplace culture, how to build trust and safety, and the importance of providing space for people.</p>",
```

Comment:
Oops! I didn't notice this until after pointing at buzzsprout for this command, but this block wouldn't have displayed very nicely in slack.